packages/astro/src/core/build/util.ts

```diff
@@ -36,3 +36,19 @@ export function i18nHasFallback(config: AstroConfig): boolean {
 
 	return false;
 }
+
+export function encodeName(name: string): string {
```

Comment:
We should at least explain what kind of encoding this function does. Or maybe an example.